uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -439,6 +460,7 @@ function _hrcore_createHelpMenu(&$menu) {
   _hrcore_civix_insert_navigation_menu($menu, '', [
     'name' => ts('Help'),
     'permission' => 'access CiviCRM',
+    'weight' => _hrcore_getMaxMenuWeight($menu) + 1
```

Comment:
Can you add a comma to the end of all these new items, it helps in the future when reviewing diffs